packages/context/src/__tests__/unit/binding.unit.ts

```diff
@@ -191,6 +191,57 @@ describe('Binding', () => {
     });
   });
 
+  describe('toAlias(bindingKeyWithPath)', async () => {
+    it('binds to another binding with sync value', () => {
+      ctx.bind('parent.options').to({child: {disabled: true}});
+      ctx.bind('child.options').toAlias('parent.options#child');
```

Comment:
Does it matter if I bind to alias first 
`ctx.bind('child.options').toAlias('parent.options#child');`
then 
`ctx.bind('parent.options').to({child: {disabled: true}});`
?

Reply:
It does NOT matter.

Reply:
I added a test for that.